internal/services/compute/orchestrated_virtual_machine_scale_set_resource.go

```diff
@@ -674,13 +674,10 @@ func resourceOrchestratedVirtualMachineScaleSetCreate(d *pluginsdk.ResourceData,
 			}
 		}
 
-		virtualMachineProfile.OsProfile = vmssOsProfile
-	}
+		vmssOsProfile.AllowExtensionOperations = pointer.To(extensionOperationsEnabled)
```

Comment:
osType isn't used here, this doesn't seem right to me... why are we defining osType above, but we don't use it in the code anywhere?